### ISSUE TITLE
`monophyly_grip` setting does not work

### ISSUE THREAD

**Issue by xrotwang** [lmaurits/BEASTling#29]
Setting `monophyly_grip` to `loose` results in the following exception: 

```
   xml = beastling.beastxml.BeastXml(config)
  File "/home/robert/venvs/beastling/BEASTling/beastling/beastxml.py", line 32, in __init__
    self.build_xml()
  File "/home/robert/venvs/beastling/BEASTling/beastling/beastxml.py", line 110, in build_xml
    self.add_prior()
  File "/home/robert/venvs/beastling/BEASTling/beastling/beastxml.py", line 140, in add_prior
    attribs["newick"] = self.make_monophyly_newick(glotto_iso_langs)
  File "/home/robert/venvs/beastling/BEASTling/beastling/beastxml.py", line 294, in make_monophyly_newick
    struct = self.make_loose_monophyly_structure(langs)
  File "/home/robert/venvs/beastling/BEASTling/beastling/beastxml.py", line 282, in make_loose_monophyly_structure
    return [[l for l in langs if point in self.config.classifications[l.lower()] ] for point in points]
TypeError: 'bool' object is not iterable
```

**Comment by lmaurits** [comment]
Unsurprising!  The extended monophyly stuff is quite neglected it.  I wrote it quickly for some one-off application a long time ago and have not touched it sense.  It has zero test coverage and is undocumented.  I suppose we should aim to fix this for 1.1.0.  I will look into it tomorrow.

**Comment by lmaurits** [comment]
`monophyly_grip` is now ignored and is undocumented.  `monophyly_direction` is not ignored, is documented, and seems to work in the way that `monophyly_grip` was intended to.
